src/components/BottomBarTitle.vue

```diff
@@ -38,18 +38,37 @@ export default {
     filter: brightness(95%);
     transition: all 0.2s ease;
   }
+
+  .bottombar-square-title {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
 }
 
 .expandedBottomBarTitle{
     position: fixed;
     bottom: 16.25rem;
 }
 
-@media only screen and (max-width: 1347px) {
+@media only screen and (max-width: 976px) {
+  .bottombartitle {
+    left: 25.5rem;
+    width: calc(100vw - 25.5rem);
+  }
+}
+
+@media only screen and (max-width: 878px) {
```

Comment:
Should this be 976 px like the other ones?

Reply:
Here it should be 878px because the bottom bar title width changes drastically from the 976px breakpoint.

At the 979px breakpoint: To ensure that we can have the requirements bar show for as small of a screenwidth as possible, the requirements bar width is decreased and so the bottom bar title has to move left by the decreased width amount. 

At the 878px breakpoint: The requirements bar is hidden by default and becomes toggleable so the bottom bar title width has to be 100%.